server/app/views/BaseHtmlView.java

```diff
@@ -77,12 +82,14 @@ protected static ButtonTag redirectButton(String id, String text, String redirec
    *
    * @return The element itself.
    */
+  // TODO(#4657): remove this method.
   protected static <T extends Tag> T asRedirectElement(T element, String redirectUrl) {
```

Comment:
Heads up this is also called on a non-button object [here](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/app/views/admin/questions/QuestionsListView.java?L327)